rules/windows/builtin/security/win_security_susp_failed_logons_explicit_credentials.yml

```diff
@@ -1,12 +1,12 @@
-title: Multiple Users Attempting To Authenticate Using Explicit Credentials
+title: Password Spraying via Explicit Credentials
```

Comment:
As we filter Computer objects we can keep the "users", no ?

Reply:
The filtering for computer objects is at the source, not the target of the attack. Computer account passwords are normally too long and random to guess so I think it goes without saying that the target of the password spray is users and not computer accounts. However, you could change it to:
`Frequent Authentication Attempts to Multiple Users via Explicit Credentials`
`Frequent Authentication Attempts to Multiple Users from a Single User Account via Explicit Credentials`
`Possible Password Spraying Against Multiple Users via Explicit Credentials`
etc...
However, I think the shorter `Password Spraying via Explicit Credentials` is easy enough to understand and more concise.